clients/src/test/java/org/apache/kafka/clients/consumer/internals/HeartbeatRequestManagerTest.java

```diff
@@ -152,6 +152,34 @@ public void cleanup() {
         }
     }
 
+    @Test
+    public void testHeartBeatRequestStateToStringBase() {
+        final long retryBackoffMs = 100;
+        final long retryBackoffMaxMs = 1000;
+        LogContext logContext = new LogContext();
+        HeartbeatRequestState heartbeatRequestState1 = new HeartbeatRequestState(
+                logContext,
+                time,
+                10,
+                retryBackoffMs,
+                retryBackoffMaxMs,
+                .2
+        );
+
+        RequestState requestState = new RequestState(
+                logContext,
+                "org.apache.kafka.clients.consumer.internals.HeartbeatRequestManager$HeartbeatRequestState",
```

Comment:
Perhaps we could make `HeartbeatRequestState` package-protected, then we could do this:

```suggestion
                HeartbeatRequestManager.HeartbeatRequestState.class.getName(),
```